.github/workflows/test.yml

```diff
@@ -54,6 +54,9 @@ jobs:
           sudo apt update
           sudo apt install -y task-spooler time
           tsp -S 48
+          curl https://www.earthbyte.org/download/9079/ -o ./pygplates_0.36.0_py310_ubuntu-22.04.deb
+          sudo apt install -y --fix-broken ./pygplates_0.36.0_py310_ubuntu-22.04.deb
+          ln -s /usr/lib/pygplates.so /home/firedrake/firedrake/lib/python3.10/site-packages/
       - uses: actions/checkout@v4
 
```

Comment:
Just tagging @angus-g to check the above installation design!

Reply:
I think this is reasonable (quite close to what they do in the [upstream Dockerfile](https://github.com/GPlates/dockerfile/blob/main/Dockerfile.pygplates-py3-base-jammy-amd64)), although I don't think we need `--fix-broken`?

Reply:
eeee ...  This is I think what I pulled from Angus' branch?  Do you need me to change this?

Reply:
Oh, weird! Past me must've put it there for a reason. Can't really hurt, so this is fine.

Reply:
Although having said that, I might look into caching that file so we don't hit their server all the time...